### PR TITLE
REQ-03: Name validation

### PR DESCRIPTION
**Pull request by oleksandr-sydorskyi**
The Task was: To improve the code by implementing validation of the first and last name fields. 
According to the task, methods isNameValid() and getValidName() were added. 
The isNameValid method checks if the provided name matches the defined name pattern.
The getValidName method prompts the user for a name input and ensures validity.
The buildClient method was modified. The getValidName() method assigns valid values to fields firstName and lastName.

### CODE REVIEW

**Review by KaterynaMir — APPROVED** [review]
As far as I see the code is fine and can be merged ;) May be it is worth to add a prompt hint for the user on what name is considered as valid.

**Comment by vini607** [comment]
The code is well written and the feature is needed immediately. I agree to use the commit in the master branch of the next release

File: src/main/java/com/magicvet/Main.java [hunk 1]
```diff
@@ -9,6 +9,8 @@ public class Main {
     static String PASSWORD = "default";
```
Comment (vini607) [inline src/main/java/com/magicvet/Main.java:9]: the password must be made private

File: src/main/java/com/magicvet/Main.java [hunk 2]
```diff
@@ -35,32 +53,32 @@ static void registerNewClient() {
     static Client buildClient(String email) {
```
Comment (vini607) [inline src/main/java/com/magicvet/Main.java:53]: make a separate client class

**Review by vini607 — COMMENTED** [review]
as possible improvements in future releases

File: src/main/java/com/magicvet/Main.java (hunk 1 above)
Comment (oleksandr-sydorskyi) [reply src/main/java/com/magicvet/Main.java:9]: I agree, but we haven't studied access modifiers yet. So I will add it in the next release.

**Pull request merged**